docker-compose.yml

```diff
@@ -34,13 +34,13 @@ services:
 
   cache:
     image: redis:4.0.10
-    ports:
-      - "6379:6379"
+    expose:
+      - "6379"
```

Comment:
Ditto with Redis. It can be useful to be able to debug with the Redis CLI locally.

Reply:
Exposed port in d714b04.